drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -411,9 +411,10 @@ function DragAndDropTemplates(configuration) {
             ];
             popup_content = h("div.popup-content", {}, have_messages ? content_items : []);
         } else {
-            popup_content = h("div.popup-content", {}, msgs.map(function(message) {
-                return h("p", {innerHTML: message.message});
-            }))
+            popup_content = h(
+                              (ctx.last_action_correct ? "div.popup-content" : "div.popup-content.popup-content-incorrect"),   
+                              {}, msgs.map(function(message) {
+                                return h("span", {innerHTML: message.message});}))
```

Comment:
Nit: this looks pretty hard to read, relative to what it was before.

Reply:
@UmanShahzad is it just a matter of formatting the code? How do you suggest I make it easier to read?

Reply:
Yeah, I just skimmed this, and wasn't taking a look at logic or functionality, but just the code readability. I'd suggest looking at pre-existing, surrounding code, including possibly some of the code in the deletion diff.